OTA-5642 update targets upload and add-uploaded options (#1829)
advancedtelematic/aktualizr: docs/ota-client-guide/modules/ROOT/pages/upload-large-binary.adoc

  --name <name-of-target> \
  --version <target-version>
----

> NOTE: To skip checking whether the file has already been added to the targets, run the command with `--force` option.

Reviewer: ```suggestion
NOTE: Not to check if the file is in targets, run the command with the `--force` option.
```
